<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>Display Area</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="placeholderText">
         <string>Enter your text here...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ReceivedLabel">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>Log Text</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutForLogTextAndButtons">
      <item>
       <widget class="QTextEdit" name="LogText">
        <property name="placeholderText">
         <string>Logs will be displayed here...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutButtons">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutForInputs">
          <item>
           <widget class="QPushButton" name="CallButton">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;; min-width: 80px; max-width: 100px;</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Call</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="AddressLayout">
            <item>
             <widget class="QLineEdit" name="IPInput">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
              <property name="placeholderText">
               <string>Enter IP Address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PortInput">
              <property name="text">
               <string>12345</string>
              </property>
              <property name="placeholderText">
               <string>Enter Port</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="StopButton">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>